### PR TITLE
Standalone usage of set_esbuild and set_mocha functions

### LINKED ISSUE #22: Allow to initialise esbuild independently of anything else
cc @hypebright.

Idea: make this exported:

```r
# Setup esbuild for JS code management
  set_esbuild()
  # Add mocha for tests
  set_mocha()
  # Ignore files/folders: srcjs, node_modules, ...
  use_build_ignore(
    c(
      "srcjs",
      "node_modules",
      "package.json",
      "package-lock.json",
      "styles",
      "esbuild.dev.json",
      "esbuild.prod.json"
    )
  )
  use_git_ignore("node_modules")
...
```

### PR DESCRIPTION
**Pull request by hypebright**
Will close #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,33 @@ build_js()
...
+
+```r
+# Setup esbuild for JS code management
+set_esbuild()
+
+# Add mocha for tests
+set_mocha()
+
+# Ignore files/folders: srcjs, node_modules, ...
+usethis::use_build_ignore(
```
Comment (DivadNojnarg) [inline README.md:80]: Do we want to have this within `set_esbuild()` since all those files are the due to esbuild setup?

File: README.md [hunk 2]
```diff
@@ -65,6 +65,33 @@ build_js()
...
+    "node_modules",
+    "package.json",
+    "package-lock.json",
+    "styles",
+    "esbuild.dev.json",
+    "esbuild.prod.json"
+  )
+)
+
+usethis::use_git_ignore("node_modules")
```
Comment (DivadNojnarg) [inline README.md:92]: same comment.

**Review by DivadNojnarg — COMMENTED** [review]
I left 2 little comments about ignoring files within set_esbuild.

File: README.md (hunk 1 above)
Comment (hypebright) [reply README.md]: Good point, makes sense. Resolved in latest commits.

**Comment by hypebright** [comment]
moved to `set_esbuild()` as per your suggestion

**Review by DivadNojnarg — APPROVED** [review]
LGTM thanks!

**Pull request merged**